lib/bandit/http1/adapter.ex

```diff
@@ -367,12 +373,28 @@ defmodule Bandit.HTTP1.Adapter do
     415 => "Unsupported Media Type",
     416 => "Requested range not satisfiable",
     417 => "Expectation Failed",
+    418 => "I'm a teapot",
+    421 => "Misdirected Request",
+    422 => "Unprocessable Entity",
+    423 => "Locked",
+    424 => "Failed Dependency",
+    425 => "Too Early",
```

Comment:
Not in 9110

Reply:
[RFC 8470](https://datatracker.ietf.org/doc/html/rfc8470#section-5.2)